src/cli/config-flags.ts

```diff
@@ -213,13 +213,25 @@ export const CLI_FLAG_ALIASES: AliasMap = {
   h: 'help',
   p: 'port',
   v: 'version',
+
+  // JEST SPECIFIC CLI FLAGS
+  // these are defined in
+  // https://github.com/facebook/jest/blob/4156f86/packages/jest-cli/src/args.ts
```

Comment:
I pulled these out of this file by running the following (hacky!) script:

```js
const { readFileSync } = require('fs');

// this is a commonjs module, so the stuff in it will be stuck on `exports`
eval(String(readFileSync('./node_modules/jest-cli/build/cli/args.js')));

const aliases = {}

Object.entries(exports.options).forEach(([k,v]) => {
  if (typeof v.alias === 'string') {
    aliases[v.alias] = k
  }
})

console.log(JSON.stringify(aliases))
```

Reply:
(in the Stencil repo, but this would work in any npm project where Jest is installed)